subsys/mgmt/mcumgr/transport/Kconfig.bluetooth

```diff
@@ -92,4 +91,12 @@ config MCUMGR_TRANSPORT_BT_CONN_PARAM_CONTROL_RETRY_TIME
 
 endif # MCUMGR_TRASNPORT_BT_CONN_PARAM_CONTROL
 
+config MCUMGR_TRANSPORT_BT_DYNAMIC_SVC_REGISTRATION
+	bool "Register SMP service at runtime"
+	select BT_GATT_DYNAMIC_DB
+	default y
+	help
+	  When enabled, the SMP service must be registred at runtime.
```

Comment:
`is automatically registered during boot and then can be dynamically registered/unregistered using dedicated API.`
(application actually dose not need to register the service on boot)